integration-tests/actions/actions.go

```diff
@@ -609,29 +610,48 @@ func TrackForwarder(
 		Msg("Forwarder tracked")
 }
 
-// DeployOCRv2Contracts deploys a number of OCRv2 contracts and configures them with defaults
-func DeployOCRv2Contracts(
+// SetupOCRv2Contracts deploys a number of OCRv2 contracts and configures them with defaults
+func SetupOCRv2Contracts(
 	l zerolog.Logger,
 	seth *seth.Client,
-	numberOfContracts int,
+	ocrContractsConfig ocr.OffChainAggregatorsConfig,
```

Comment:
@Tofel I assume no secrets in this config? If yes, use test secrets https://github.com/smartcontractkit/chainlink-testing-framework/blob/main/config/README.md#test-secrets

Reply:
no test secrets there, it's just contract addresses